README.md

```diff
@@ -83,6 +83,10 @@ Similarly, a `ktlintFormat` task has been added that formats all of the source s
 If project has subprojects - plugin also adds two meta tasks `ktlintCheck` and `ktlintFormat` to the root project that
 triggers related tasks in subprojects.
 
+Another task added: `ktlintApplyToIdea`. It is always added only to the
+root project. Task generates IntelliJ IDEA (or Android Studio) Kotlin
+style files in `.idea/` folder. **Note** that this task will *overwrite*
+existing ones style files.
```

Comment:
> **Note** that this task will *overwrite* existing ones style files.

Gramatically, that's weird.

Reply:
English is not my native language. What is the right wording? *will replace*?

Reply:
Perhaps:
> Note that this task will overwrite any existing style file.

Unless there is a singular file that it could overwritten, then it might be:

> Note that this task will overwrite the existing style file.

Reply:
fixed the wording